.babelrc

```diff
@@ -21,6 +21,7 @@
       "presets": ["env", "stage-2"],
       "plugins": [ 
         "istanbul",
+        "dynamic-import-node",
```

Comment:
This converts `import()` syntax to `require()` statements when running tests in Node.